examples/react/module-api-restful/src/components/user-view.tsx

```diff
@@ -1,10 +1,11 @@
 import { useUser } from "../hooks/use-user";
 
-export function UserView() {
-  const query = useUser();
+export function UserView({ id }: { id: string }) {
```

Comment:
[IMO] I think it would be cleaner to extract the props type from here. I understand that this is supposed to be only a simple example, but considering that we should also demonstrate good practices and clean code, what do you think?

```ts
interface UserViewProps {
  id: string
}

export function UserView({ id }: UserViewProps) {
...
```